cmd/main.go

```diff
@@ -116,8 +117,22 @@ func (c Configuration) consumers(db *pkg.AgentDB) (consumers pkg.BaseConsumers)
 			consumers = append(consumers, &pkg.BaseConsumer{AgentDB: db, ParserLoader: state})
 		}
 	}
+	if len(c.Consumers.Sudoers) > 0 {
+		//get list of files to watch
+		sudoersFiles := c.getListOfFiles(fs, "sudoers")
+		for _, sudoersFile := range sudoersFiles {
+			if !c.fileBelongsToExclusionList(sudoersFile.File) {
```

Comment:
put `fileBelongsToExclusionList` in the `getListOfFiles`

Reply:
The function is getListOfFiles that had this logic, is removed. Keeping this here, with additional changes.